docs/docs/rest-api/public/api/v2/groups.raml

```diff
@@ -191,6 +191,21 @@
       the given version.
     is: [ secured, deployable ]
 
+  patch:
```

Comment:
[MARATHON-8638](https://jira.mesosphere.com/browse/MARATHON-8638) is not really clear whether it should be a new endpoint or part of the `PUT` call. I think a new endpoint enables us to be backward compatible so I opted for that.

Reply:
We could also only allow the path for the top-level path. This way we won't have to validate the update.

Reply:
> We could also only allow the path for the top-level path. This way we won't have to validate the update.

That sounds reasonable to me

Reply:
```
curl -X PATCH :8080/v2/groups/dev -d '{"enforceRole": "top"}'
```
seems good to me.